shell-plugin/forge.plugin.zsh

```diff
@@ -49,7 +49,7 @@ ZSH_HIGHLIGHT_HIGHLIGHTERS+=(pattern)
 # Loads the commands list only when first needed, avoiding startup cost
 function _forge_get_commands() {
     if [[ -z "$_FORGE_COMMANDS" ]]; then
-        _FORGE_COMMANDS="$(CLICOLOR_FORCE=0 $_FORGE_BIN list commands --porcelain 2>/dev/null)"
+        _FORGE_COMMANDS="$(CLICOLOR_FORCE=0 $_FORGE_BIN list commands --porcelain 2>/dev/null | sort -k1,1)"
```

Comment:
Can be done in `rust`. We have a sort by command on porcelain outputs.

Reply:
Sorting for porcelain can be done in rust, because we can write tests in rust code to ensure the output is always sorted by some key.